src/main/java/uk/gov/hmcts/reform/unspec/service/docmosis/sealedclaim/SealedClaimFormGenerator.java

```diff
@@ -0,0 +1,109 @@
+package uk.gov.hmcts.reform.unspec.service.docmosis.sealedclaim;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.reform.unspec.model.Address;
+import uk.gov.hmcts.reform.unspec.model.CaseData;
+import uk.gov.hmcts.reform.unspec.model.Party;
+import uk.gov.hmcts.reform.unspec.model.docmosis.DocmosisDocument;
+import uk.gov.hmcts.reform.unspec.model.docmosis.sealedclaim.Claimant;
+import uk.gov.hmcts.reform.unspec.model.docmosis.sealedclaim.Defendant;
+import uk.gov.hmcts.reform.unspec.model.docmosis.sealedclaim.Representative;
+import uk.gov.hmcts.reform.unspec.model.docmosis.sealedclaim.SealedClaimForm;
+import uk.gov.hmcts.reform.unspec.model.documents.CaseDocument;
+import uk.gov.hmcts.reform.unspec.model.documents.DocumentType;
+import uk.gov.hmcts.reform.unspec.model.documents.PDF;
+import uk.gov.hmcts.reform.unspec.service.docmosis.DocumentGeneratorService;
+import uk.gov.hmcts.reform.unspec.service.docmosis.TemplateDataGenerator;
+import uk.gov.hmcts.reform.unspec.service.documentmanagement.DocumentManagementService;
+
+import java.time.LocalDate;
+import java.util.List;
+
+import static uk.gov.hmcts.reform.unspec.service.docmosis.DocmosisTemplates.N1;
+import static uk.gov.hmcts.reform.unspec.utils.PartyNameUtils.getPartyNameBasedOnType;
+
+@Service
+public class SealedClaimFormGenerator extends TemplateDataGenerator<SealedClaimForm> {
+
+    public static final String TEMP_CLAIM_DETAILS = "The claimant seeks compensation from injuries and losses arising"
+        + " from a road traffic accident which occurred on 1st July 2017 as a result of the negligence of the first "
+        + "defendant.The claimant seeks compensation from injuries and losses arising from a road traffic accident "
+        + "which occurred on 1st July 2017 as a result of the negligence of the first defendant."; //TODO
+
+    private static final Representative TEMP_REPRESENTATIVE = Representative.builder()
+        .contactName("MiguelSpooner")
+        .dxAddress("DX 751Newport")
+        .organisationName("DBE Law")
+        .phoneNumber("0800 206 1592")
+        .emailAddress("jim.smith@slatergordon.com")
+        .serviceAddress(Address.builder()
+                            .addressLine1("AdmiralHouse")
+                            .addressLine2("Queensway")
+                            .postTown("Newport")
+                            .postCode("NP204AG")
+                            .build())
+        .build(); //TODO
+    public static final String REFERENCE_NUMBER = "000LR095"; //TODO
+    public static final String CASE_NAME = "SamClark v AlexRichards"; //TODO
+    private final DocumentManagementService documentManagementService;
+    private final DocumentGeneratorService documentGeneratorService;
+
+    @Autowired
+    public SealedClaimFormGenerator(DocumentManagementService documentManagementService,
```

Comment:
can be replaced with `@RequiredArgsConstructor(onConstructor = @__(@Autowired))` annotation